### PR TITLE
Write release doc for RM.

### PR DESCRIPTION
**Pull request by wu-sheng**
@wuwen5 Could you review this doc?

The doc was missed, and I barely put them together.

### CODE REVIEW

File: RELEASING.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+ * sha512 checksums
+   - xxx  skywalking-nginx-lua-${RELEASE_VERSION}-src.tgz
+
+Release Tag :
+
+ * v${RELEASE_VERSION}
+
+Release CommitID :
+
+ * https://dist.apache.org/repos/dist/dev/skywalking/nginx-lua/${RELEASE_VERSION}/
```
Comment (wu-sheng) [inline RELEASING.md]:
```suggestion
 * https://dist.apache.org/repos/dist/dev/skywalking/nginx-lua/${COMMIT_ID}
```

File: RELEASING.md [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+[ ] +1 Release this package.
+[ ] +0 No opinion.
+[ ] -1 Do not release this package because....
+```
+
+7. If the vote passed with at least +1 binding(s) and more +1 binding(s) than -1 binding(s), the vote pass.
+
+8. Upload the rockspec to LuaRocks
+
+> luarocks upload skywalking-nginx-lua-${RELEASE_VERSION}-0.rockspec
```
Comment (wu-sheng) [inline RELEASING.md]:
```suggestion
// As a release manager, you could ask the luarocks account in the private mail list, and get this API key to upload the new release.

> luarocks upload skywalking-nginx-lua-${RELEASE_VERSION}-0.rockspec --api-key=xxx
```

**Review by wankai123 — APPROVED** [review]

**Comment by wu-sheng** [comment]
I tested this, and have this out on the luarocks.

https://luarocks.org/modules/apache-skywalking/skywalking-nginx-lua/1.0.0-0

File: RELEASING.md [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+
+1. Prepare the new `.rockspec` file for the new release. Ref previous and latest files in [here](./rockspec/).
+2. Update the [changelogs](CHANGES.md) for the upcoming release.
+3. Tag through git or GitHub.
+4. Build the source tars with ASC sign and SHA512
+
+```shell
+> export RELEASE_VERSION=x.y.z
+
+> tar czf skywalking-nginx-lua-${RELEASE_VERSION}-src.tgz \
```
Comment (wuwen5) [inline RELEASING.md:16]: There is already a [release.md](https://github.com/apache/skywalking-nginx-lua/blob/master/release.md) file, Do we recommend using `make release-src` here ?

Comment (wu-sheng) [reply RELEASING.md:16]: Oops, I missed that. I am going to merge these two.

**Pull request merged**